### PR TITLE
Dm dev/1395

### LINKED ISSUE #1395: [FEATURE] Добавить большее кол-во якорей в СС (скальперский стакан)
### Опишите проблему, которую решает ваша фича

Недостаточное кол-во якорей

### Опишите предлагаемое вами решение

Необходимо добавить кол-во якорей до 16 штук.

### Опишите альтернативные решения, о которых вы думали

_No response_

### Дополнительная информация

_No response_

### Правила

- [X] Я согласен следовать правилам

### PR DESCRIPTION
**Pull request by agushasok**
Fixes #1395 

# Description

add more color labels

# Testing

see in linked issue

# Risk

No major changes

# Do you agree with those?
- [] I agree to follow the [Contributing guidelines](https://github.com/alor-broker/Astras-Trading-UI/blob/master/CONTRIBUTING.md)
- [] I agree to follow the terms of the [Code of Conduct](https://github.com/alor-broker/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: src/app/modules/terminal-settings/components/general-settings-form/general-settings-form.component.ts
Comment (agushasok) [inline src/app/modules/terminal-settings/components/general-settings-form/general-settings-form.component.ts:1]: вынести логику лейблов в отдельный компонент

File: src/app/modules/terminal-settings/components/general-settings-form/general-settings-form.component.ts [hunk 1]
```diff
@@ -144,4 +156,77 @@ export class GeneralSettingsFormComponent extends ControlValueAccessorBaseCompon
...
+  addPredefinedLabels(): void {
+    if (this.form == null) {
+      return;
+    }
+
+    const badgesColorsFormValue = this.form.controls.badgesColors.value as string[];
+
+    this.form.controls.badgesColors.setValue([
+      ...badgesColorsFormValue,
+      ...[...instrumentsBadges, ...additionalInstrumentsBadges].filter(b => !badgesColorsFormValue.includes(b)).slice(0, 2)
```
Comment (agushasok) [inline src/app/modules/terminal-settings/components/general-settings-form/general-settings-form.component.ts:188]: вынести за пределы setValue

File: src/app/modules/terminal-settings/components/general-settings-form/general-settings-form.component.ts [hunk 2]
```diff
@@ -144,4 +156,77 @@ export class GeneralSettingsFormComponent extends ControlValueAccessorBaseCompon
...
+  }
+
+  removeBadgeColor(e: MouseEvent, color: string): void {
+    if (this.form == null) {
+      return;
+    }
+
+    e.stopPropagation();
+
+    this.form!.controls.badgesColors.setValue(
```
Comment (agushasok) [inline src/app/modules/terminal-settings/components/general-settings-form/general-settings-form.component.ts:199]: убрать "!"

File: src/assets/i18n/terminal-settings/general-settings-form/ru.json [hunk 3]
```diff
@@ -27,6 +27,8 @@
   "badgesBindHint1": "Выбрав инструмент в виджете с бейджем определённого цвета, поменяется выбранный инструмент во всех виджетах с бейджем этого же цвета",
```
Comment (agushasok) [inline src/assets/i18n/terminal-settings/general-settings-form/ru.json:27]: заенить везде бейджи на лейблы, а labels  на badges

**Review by PilgrimViis — APPROVED** [review]

**Pull request merged**